ui/src/app/views/dataset-page/component.tsx

```diff
@@ -545,16 +573,24 @@ const DataSetPage = withCurrentWorkspace()(class extends React.Component<Props,
         <div style={{backgroundColor: 'white', border: '1px solid #E5E5E5'}}>
           <div style={{...styles.selectBoxHeader, display: 'flex', flexDirection: 'row',
             position: 'relative'}}>
+            <div style={styles.previewDataHeader}>
+              Preview Data Set
+            </div>
+            {!defaultPreviewView && <Clickable data-test-id='preview-icon'
+                                               onClick={() => this.getPreviewList()}
+                                               style={styles.refreshIcon}>
+              <ClrIcon style={{fill: colors.gray[7], marginLeft: '0.1rem', marginTop: '-1rem'}}
```

Comment:
Can you use align-self rather than a negative marginTop? Or maybe the clickable can do it? Just want to make sure we are using relative rather than absolute styling when we can.

Reply:
I was not able to align the actual icon to the middle of the button. I tried align-items and vertical align. We can take a look at it together?